scrap.py

```diff
@@ -0,0 +1,68 @@
+import requests
+from bs4 import BeautifulSoup
+import json
+
+class Scrapping():
+    
+    def __init__(self):
+      self.url = 'https://simpsonizados.me/serie/los-simpson/'
+      self.__db = 'simpson_db.json'
+
+    def get_data(self):
+      req = requests.get(self.url)
+      page = req.text
+      soup = BeautifulSoup(page, 'lxml')
+      all_episodes = soup.find_all('ul', class_='episodios')
+      result = []
+      temp = 1
+      for i in range(0, len(all_episodes)):
```

Comment:
Podes utilizar enumerate para iterar sobre la temporada y el índice a la vez.

`for season_number, season in enumerate(seasons):`